Events/2022-05-ECP22-BOF.md

```diff
@@ -0,0 +1,55 @@
+# 2022 Exascale Computing Project Community BOF Days
+
+#### Contributed by BSSw.io Team
+
+- Date: May 10 - 12, 2022
```

Comment:
We now prefer to say "Event Dates" (or singular, as appropriate)

Reply:
Thanks, I've made that change!